src/test/groovy/org/gradle/android/RelocationTest.groovy

```diff
@@ -1,286 +0,0 @@
-package org.gradle.android
-
-import com.google.common.collect.ImmutableMap
-import org.gradle.testkit.runner.BuildResult
-import org.gradle.testkit.runner.TaskOutcome
-import org.gradle.util.GradleVersion
-import org.gradle.util.VersionNumber
-import spock.lang.Unroll
-
-import static org.gradle.android.Versions.android
-import static org.gradle.android.Versions.gradle
-import static org.gradle.testkit.runner.TaskOutcome.FROM_CACHE
-import static org.gradle.testkit.runner.TaskOutcome.NO_SOURCE
-import static org.gradle.testkit.runner.TaskOutcome.SUCCESS
-import static org.gradle.testkit.runner.TaskOutcome.UP_TO_DATE
-
-class RelocationTest extends AbstractTest {
```

Comment:
Why remove this test? This used to check if the workarounds actually worked.

Reply:
I would add one more thing to any test we end up having: explicitly check for deprecation warnings emitted. Just to make sure we don't end up using some deprecated APIs unintentionally.